components/integrations/bots/bots.tsx

```diff
@@ -1,98 +1,108 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
+import {Bot as BotType} from 'mattermost-redux/types/bots';
+import {Dictionary, RelationOneToOne} from 'mattermost-redux/types/utilities';
+import {UserProfile, UserAccessToken} from 'mattermost-redux/types/users';
+import {ActionResult} from 'mattermost-redux/types/actions';
+import {Team} from 'mattermost-redux/types/teams';
 
 import {getSiteURL} from 'utils/url';
-
 import * as Utils from 'utils/utils.jsx';
 import BackstageList from 'components/backstage/components/backstage_list.jsx';
 import Constants from 'utils/constants';
 import FormattedMarkdownMessage from 'components/formatted_markdown_message';
 
-import Bot, {matchesFilter} from './bot.jsx';
+import Bot, {matchesFilter} from './bot';
+
+type Props = {
+
+    /**
+    *  Map from botUserId to bot.
+    */
+    bots: Dictionary<BotType>;
```

Comment:
I believe we're trying to move away from these helper types and just use standard TypeScript helper types. So this can just be `Record<string, BotType>`